bedrock/settings/base.py

```diff
@@ -241,10 +241,11 @@ def lazy_langs():
 
 # TODO create these files by porting from their .lang equivalents
 FLUENT_DEFAULT_FILES = ['brands', 'navigation', 'footer', 'download_button']
+FLUENT_DEFAULT_PERCENT_REQUIRED = config('FLUENT_DEFAULT_PERCENT_REQUIRED', default='80', parser=int)
 FLUENT_REPO = config('FLUENT_REPO', default='https://github.com/mozmeao/www-l10n')
 FLUENT_REPO_PATH = GIT_REPOS_PATH / 'www-l10n'
 FLUENT_LOCAL_PATH = ROOT_PATH / 'l10n'
-FLUENT_L10N_TEAM_REPO = config('FLUENT_REPO', default='https://github.com/mozmeao/www-l10n')
+FLUENT_L10N_TEAM_REPO = config('FLUENT_REPO', default='https://github.com/mozilla-l10n/www-l10n')
 FLUENT_L10N_TEAM_REPO_PATH = GIT_REPOS_PATH / 'l10n-team'
```

Comment:
When I try running the command I get an error because this path doesn't exist in my git-repos directory?

```
(venv) agibson-42230:bedrock agibson$ ./manage.py process_ftl
INFO 2020-02-14 02:49:26,513 base Raven is not configured (logging is disabled). Please see the documentation for more information.
Updated .ftl files
Traceback (most recent call last):
  File "./manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/Users/agibson/Documents/GitHub/bedrock/venv/lib/python3.7/site-packages/django/core/management/__init__.py", line 381, in execute_from_command_line
    utility.execute()
  File "/Users/agibson/Documents/GitHub/bedrock/venv/lib/python3.7/site-packages/django/core/management/__init__.py", line 375, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/Users/agibson/Documents/GitHub/bedrock/venv/lib/python3.7/site-packages/django/core/management/base.py", line 323, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/Users/agibson/Documents/GitHub/bedrock/venv/lib/python3.7/site-packages/django/core/management/base.py", line 364, in execute
    output = self.handle(*args, **options)
  File "/Users/agibson/Documents/GitHub/bedrock/lib/l10n_utils/management/commands/process_ftl.py", line 47, in handle
    self.update_l10n_team_files()
  File "/Users/agibson/Documents/GitHub/bedrock/lib/l10n_utils/management/commands/process_ftl.py", line 57, in update_l10n_team_files
    self.l10n_repo.clean()
  File "/Users/agibson/Documents/GitHub/bedrock/bedrock/utils/git.py", line 137, in clean
    self.git('clean', '-fd')
  File "/Users/agibson/Documents/GitHub/bedrock/bedrock/utils/git.py", line 37, in git
    os.chdir(self.path_str)
FileNotFoundError: [Errno 2] No such file or directory: '/Users/agibson/Documents/GitHub/bedrock/git-repos/l10n-team'
```

I've tried running `l10n_update` first but no dice. Any ideas?

Reply:
Doh! Yeah. In the command I put a call to clean() before update(). This doesn’t work on first run. You can move the clean() call to after update for now. I’ll fix it for real soon.

Reply:
It worked locally when I tried this 👍

Reply:
I have now pushed a fix for this.